src/ott/solvers/linear/univariate.py

```diff
@@ -12,134 +12,247 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Callable, Literal, Optional
+from typing import NamedTuple, Optional, Union
 
 import jax
 import jax.numpy as jnp
 
-from ott.geometry import costs
+from ott.geometry import costs, pointcloud
+from ott.problems.linear import linear_problem
 
-__all__ = ["UnivariateSolver"]
+__all__ = ["UnivariateOutput", "UnivariateSolver"]
+
+
+class UnivariateOutput(NamedTuple):  # noqa: D101
```

Comment:
Missing documentation for the class/properties.

Reply:
thanks for the catch